scenarios/mnist/contract/contract.json

```diff
@@ -0,0 +1,93 @@
+{
+  "id": "f4f72a88-bab1-11ed-afa1-0242ac120002",
+  "schemaVersion": "0.1",
+  "startTime": "2023-03-14T00:00:00.000Z",
+  "expiryTime": "2024-03-14T00:00:00.000Z",
+  "tdc" : "",
+  "tdps" : [],
+  "ccrp": "did:web:ccrprovider.github.io",
+  "datasets": [
+    {
+      "id" : "19517ba8-bab8-11ed-afa1-0242ac120002",
+      "name": "mnist_1",
+      "url" : "https://ccrcontainer.blob.core.windows.net/mnist_1/data.img",
+      "provider": "",
+      "key" : {
+        "type": "azure",
+        "properties": {
+          "kid": "MNIST_1_FilesystemEncryptionKey",
```

Comment:
kid is not consistent as in the previous PR

Reply:
making the kid consistent "MNIST_1_FilesystemEncryptionKey"